src/Robo/Commands/Deploy/DeployCommand.php

```diff
@@ -488,10 +488,13 @@ protected function composerInstall() {
     if ($this->ignorePlatformReqs) {
       $command .= ' --ignore-platform-reqs';
     }
-    $this->taskExecStack()->exec($command)
+    $execution_result = $this->taskExecStack()->exec($command)
       ->stopOnFail()
       ->dir($this->deployDir)
       ->run();
+    if(!$execution_result->wasSuccessful()) {
+      throw new BltException("Composer Install failed, please review and check your composer.json file.");
```

Comment:
Is there a way to pass the $exception_result message as the exception?

Reply:
I'm not sure if that would look the best, due to the error being displayed already above the exception output. Also, it does appear that this is the suggested format and there might be a re-implementation on how these exceptions are going to be captured per danepowell last comment.

Reply:
Ok, let's adjust messaging to account for the scenario where the composer install failed due to network issue.

What about:

```
Composer install failed, please check the output for details.
```